app/models/form/lettings/questions/address_line1.rb

```diff
@@ -8,7 +8,7 @@ def initialize(id, hsh, page)
     @plain_label = true
     @check_answer_label = "Address lines 1 and 2"
     @disable_clearing_if_not_routed_or_dynamic_answer_options = true
-    @question_number = 12
+    @question_number = QUESTION_NUMBER_FROM_YEAR.fetch(form.start_date.year, QUESTION_NUMBER_FROM_YEAR.max_by { |k, _v| k }.last)
```

Comment:
I've been trying to come up with a clearer way of getting the "default" value here - I wonder if `QUESTION_NUMBER_FROM_YEAR[QUESTION_NUMBER_FROM_YEAR.keys.max]` is a more intuitive way of getting the same thing?

Reply:
`.last` seems like a confusing way to get the value of a hash pair is all